description: MADV_FREE is newly defined in glibc 2.24, but currently unsupported by webkit, so undefine it
author: Allan Sandfeld Jensen <allan.jensen@theqtcompany.com>
origin: http://code.qt.io/cgit/qt/qtwebengine-chromium.git/commit/?h=49-based&id=b12ffcd411d4776f7120ccecb3be34344d930d2b

--- a/third_party/WebKit/Source/wtf/allocator/PageAllocator.cpp
+++ b/third_party/WebKit/Source/wtf/allocator/PageAllocator.cpp
@@ -41,6 +41,10 @@
 #include <errno.h>
 #include <sys/mman.h>
 
+#if OS(LINUX) && defined(MADV_FREE)
+#undef MADV_FREE
+#endif
+
 #ifndef MADV_FREE
 #define MADV_FREE MADV_DONTNEED
 #endif
